core/server/worker/src/main/java/alluxio/worker/block/annotator/DefaultBlockIterator.java

```diff
@@ -116,7 +116,8 @@ private void blockUpdated(long blockId, BlockStoreLocation location) {
     }
 
     if (LOG.isDebugEnabled()) {
-      LOG.debug("Block:{} updated at location:{}. Size:{}", blockId, location, sortedSet.size());
+      LOG.debug("Block:{} updated at {} with {} blocks.",
```

Comment:
Can this fit on one line? (applies in general)

Reply:
It usually leaves the last or two on the second line, that's why I moved all to the same line.

Reply:
This and one other fit indeed. However I did this mostly when the last or two parameter spilled to the second line.